packages/react-hook-form-macaw/package.json

```diff
@@ -8,6 +8,7 @@
     "storybook": "storybook dev -p 6006"
   },
   "dependencies": {
+    "@saleor/app-sdk": "0.40.0",
```

Comment:
Is it intentional? Don't think we are using the SDK in bindings.

Reply:
good catch, it was accidentally installed with `pnpm -r` flag